.travis.yml

```diff
@@ -52,15 +52,19 @@ jobs:
       arch: arm64
       script:
       - docker build -t openhabian/bats-openhabian -f Dockerfile.openhabian-aarch64-native .
-      - docker run --name "bats-inst-tests" -i openhabian/bats-openhabian bash -c '/usr/local/bin/bats -r -f "installation-." .'
-      - docker run --name "bats-dest-tests" -i openhabian/bats-openhabian bash -c '/usr/local/bin/bats -r -f "destructive-." .'
+      - docker run --name "bats-inst-tests" -d openhabian/bats-openhabian;
+        docker exec -i openhabian/bats-openhabian bash -c './build.bash local-test && mv ~/.profile ~/.bash_profile && /etc/rc.local'
```

Comment:
why ? That BATS test should be built to work standalone i.e. this should not be needed

BATS tests is unit testing, build.bash is really integration testing which belongs in the next stage

Reply:
The reason being is that Java is not installed in the Virtual Machine and as such the bats test fails.

Reply:
huh? The BATS test case _itself_ should install Java, making use of the openhabian routines.
That's the whole point of unit testing.

Reply:
I'm talking about the test case here:
``` shell
@test "installation-java_exist" {
  run java -version 2>&1> /dev/null
  [ "$status" -eq 0 ]
  [[ $output == *"Zulu"* ]]
}
```
Which is run before the install routines are ever tested. Unless it is just a redundant test that needs to be removed.

Reply:
yes if it's first to run it does not make sense. Remove it.

Reply:
Ok thats what confused me. Especially since it is redone in all of the installation tests.